src/SlashCommands.js

```diff
@@ -282,6 +282,63 @@ var commands = {
             }
         }
         return reject(this.getUsage());
+    }),
+
+    // Verify a user, device, and pubkey tuple
+    verify: new Command("verify", "<userId> <deviceId> <deviceSigningKey>", function(room_id, args) {
+        if (args) {
+            var matches = args.match(/^(\S+) +(\S+) +(\S+)?$/);
```

Comment:
not sure whether it'd be more sensible to use more specific character groups, since all 3 have specific grammars